fiatdao/fiat-ui
# Localdev fork mainnet

## Testing Instructions
Follow the new instructions in the README for `Running with Localhost Network backend`

Ensure you can open a position via underlier in the UI
